invenio_rdm_records/assets/semantic-ui/js/invenio_rdm_records/src/deposit/fields/CreatibutorsField/CreatibutorsModal.js

```diff
@@ -206,10 +248,12 @@ export class CreatibutorsModal extends Component {
     } else if (identifier.scheme === "ror") {
       icon = "/static/images/ror-icon.svg";
       link = "https://ror.org/" + identifier.identifier;
+    } else if (identifier.scheme === "isni" || identifier.scheme === "grid") {
+      return <></>;
```

Comment:
this is a placeholder for identifiers that we do not have, right? are those the only missing (because below you process all the other identifiers in the else block)?

```suggestion
      return null;
```
no need to instanciate and return fragment:)

Reply:
yes, if these are handled in the else block they show up in the dropdown alongside RoR with unknown icon